tests/e2e/e2e_ibc_test.go

```diff
@@ -237,9 +237,10 @@ func (s *IntegrationTestSuite) TestIBCTokenTransfer() {
 	s.Run("send_uatom_to_chainB", func() {
 		// require the recipient account receives the IBC tokens (IBC packets ACKd)
 		var (
-			ibcStakeDenom string
 			balances      sdk.Coins
 			err           error
+			beforeBalance int64
```

Comment:
beforeBalance math.Int type ?

Reply:
IMHO is always better to use `sdk.Dec` to manage balances values so we can't lose the precision when we do math operations